common/Services/NotificationService.cs

```diff
@@ -0,0 +1,214 @@
+           .BuildNotification();
+
+        AppNotificationManager.Default.Show(toast);
+        return toast.Id != 0;
+    }
+
+    public void Initialize(StackedNotificationsBehavior notificationQueue)
+    {
+        _notificationQueue = notificationQueue;
+    }
+
+    public void HandlerNotificationActions(AppActivationArguments args)
+    {
+        if (args.Data is ToastNotificationActivatedEventArgs toastArgs)
+        {
+            try
+            {
+                if (toastArgs.Argument.Contains("action=AddUserToHyperVAdminGroup"))
+                {
+                    // Launch compmgmt.msc in powershell
+                    var psi = new ProcessStartInfo();
+                    psi.FileName = "powershell";
+                    psi.Arguments = "Start-Process compmgmt.msc -Verb RunAs";
+                    Process.Start(psi);
+                }
+            }
+            catch (Exception ex)
+            {
+                _log.Error(_componentName, $"Unable to launch computer management due to exception", ex);
+            }
+        }
+    }
+
+    public void ShowRestartNotification()
+    {
+        if (_notificationQueue != null)
+        {
+            var command = new RelayCommand(() =>
+            {
+                System.Diagnostics.Process process = new System.Diagnostics.Process();
+                System.Diagnostics.ProcessStartInfo startInfo = new System.Diagnostics.ProcessStartInfo();
+                startInfo.WindowStyle = System.Diagnostics.ProcessWindowStyle.Hidden;
+                startInfo.FileName = "cmd.exe";
+
+                // Restart the computer
+                startInfo.Arguments = "/C shutdown -r -t 0";
+                startInfo.Verb = string.Empty;
+                process.StartInfo = startInfo;
+                process.Start();
+            });
+
+            _dispatcher.EnqueueAsync(() =>
+                _notificationQueue?.Show(new Notification
+                {
+                    Title = _stringResource.GetLocalized("HyperVErrorTitle"),
+                    Message = _stringResource.GetLocalized("RestartMessage"),
+                    Severity = InfoBarSeverity.Warning,
+                    ActionButton = new Button
+                    {
+                        Content = _stringResource.GetLocalized("RestartButton"),
+                        Command = command,
+                    },
+                }));
+        }
+        else
+        {
+            _log.Error(_componentName, "Notification queue is not initialized");
+        }
+    }
+
+    public void CheckIfUserIsAHyperVAdminAndShowNotification()
+    {
+        if (!_windowsIdentityService.IsUserHyperVAdmin())
+        {
+            if (_notificationQueue != null)
+            {
+                var notification = new Notification();
+
+                var command = new RelayCommand(() =>
+                {
+                    System.Diagnostics.Process process = new System.Diagnostics.Process();
```

Comment:
I don't know why VS does not complain on initialization here (maybe because of fully qualified type name, but to be consistent with other code:
var process = new System.Diagnostics.Process();
or
System.Diagnostics.Process process = new();

Reply:
Yes, that is strange. Changed.